python/tests/runtime_aggtest/aggtst_base.py

```diff
@@ -210,6 +241,15 @@ def run_pipeline(self, pipeline_name_prefix: str, sql: str, views: list[View]):
 
             for view in views:
                 view.validate(pipeline)
+        except Exception as e:
```

Comment:
can't this conversion be done inside the sdk somewhere so other people can benefit?

Reply:
There might be some improvement to either the API or the Python SDK that will make handling of such situations cleaner. This wrapper is definitely not such an improvement. It just (hopefully) fixes the test given the API that we have today.